# Namespace to publish response topics in. 
string name
---
sensor_msgs/CameraInfo info_msg
